apps/frontend/src/app/Components/PageAndSortOptionSelect.tsx

```diff
@@ -0,0 +1,79 @@
+import { Pagination, Typography } from '@mui/material'
+import type { TFunction } from 'i18next'
+import { Trans } from 'react-i18next'
+import SortByButton from '../Components/SortByButton'
+
+type PaginationProps = {
+  count: number
+  page: number
+  onChange: (_: any, value: number) => void
+}
+
+type ShowingItemProps = {
+  numShowing: number
+  total: string
+  t: TFunction<[string, string], undefined>
+}
+
+type SortByButtonProps = {
+  sortKeys: string[]
+  value: string
+  onChange: (value: string) => void
+  ascending: boolean
+  onChangeAsc: (value: boolean) => void
+}
+
+export default function PageAndSortOptionSelect({
+  paginationProps,
+  showingTextProps,
+  displaySort = false,
+  sortByButtonProps = undefined,
+}: {
+  paginationProps: PaginationProps
```

Comment:
This is an interesting way to organize the props. I usually opt to have flatter props, but I like how you group these according to use. 
I think in this case, there are quite a lot of props, so this structure is more beneficial.